datahandling.py

```diff
@@ -298,8 +297,12 @@ def refresh_sites():
     has_more = True
     page = 1
     while has_more:
-        resp = requests.get("https://api.stackexchange.com/2.2/sites?filter=!%29Qpa1bTB_jCkeaZsqiQ8pDwI&pagesize=500&page=" + str(page) + "&key=IAkbitmze4B8KpacUfLqkw((")
-        data = json.loads(resp.text)
+        response = requests.get(
+            "https://api.stackexchange.com/2.2/sites?" +
+            "filter=!%29Qpa1bTB_jCkeaZsqiQ8pDwI&pagesize=500&page={}".format(str(page)) +
```

Comment:
If, after this is reverted, you continue using `format`, you don't need `str` here.